packages/matrix/docker/synapse/dev/homeserver.yaml

```diff
@@ -87,6 +87,9 @@ email:
   enable_notifs: true
   notif_for_new_users: false
   client_base_url: "http://localhost:4200"
-  validation_token_lifetime: "15m"
+  validation_token_lifetime: "48h"
```

Comment:
Why change to 48h?

Reply:
the ticket specifically asked for 48 hours